fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -441,6 +445,7 @@ struct TenantManagementWorkload : TestWorkload {
 				}
 				ASSERT(alreadyExists);
 				ASSERT(entry.id == tenantState.id);
+				ASSERT(entry.encrypted == tenantState.encrypted);
```

Comment:
The `check` function at the end also checks the state of the tenants, so you could add a similar assert there.